rcgen/src/sign_algo.rs

```diff
@@ -145,19 +145,17 @@ pub(crate) mod algo {
 		params: SignatureAlgorithmParams::Null,
 	};
 
-	// TODO: not really sure whether the certs we generate actually work.
-	// Both openssl and webpki reject them. It *might* be possible that openssl
-	// accepts the certificate if the key is a proper RSA-PSS key, but ring doesn't
-	// support those: https://github.com/briansmith/ring/issues/1353
+	// openssl accepts the certificate if the key is a proper RSA-PSS key, but
+	// ring doesn't support those: https://github.com/briansmith/ring/issues/1353
```

Comment:
This comment seems good as-is, but I think it would be prudent to rejig this so that the mention of there being no support in `*ring*` is part of the public rustdoc comment that makes it into documentation. Maybe something like:

```rust
	/// RSA signing with PKCS#1 2.1 RSASSA-PSS padding and SHA-256 hashing as per [RFC 4055](https://tools.ietf.org/html/rfc4055)
    ///
	/// Note: `*ring*` does not support this signature algorithm, and so it can not be used with the `crypto` feature
	/// of `rcgen` when verifying signatures using the `ring` backend.
	pub static PKCS_RSA_PSS_SHA256: SignatureAlgorithm = SignatureAlgorithm {
```

It looks like `aws-lc-rs` _does_ support these signature algorithms.

Reply:
The certificate being generated is failing here

Reply:
Have you considered making a minimal reproducer and opening an issue with aws-lc-rs? I've had good experience interacting with the maintainers.